src/context/context.service.ts

```diff
@@ -94,25 +94,15 @@ export class ContextService {
     return $('<div>').html($.html()).text().trim();
   }
 
-     setHtmlBody(
-    body: string,
-    loadAsDocument = false,
-    divId = 'konviw-ontent',
-  ): void {
+  setHtmlBody(body: string, loadAsDocument = true): void {
```

Comment:
I have kept your suggestion while set `loadAsDocument = true` by default to keep compatibility with former usage in `renderPage` and `renderSlides` and many tests.
However, it may be easier and less prone to errors just to define two utilities to load the body in the context module:
- `setHtmlBody` as it was with `loadAsDocument = true` by default
- `setBody` with `loadAsDocument = false` for the needed behaviour in the API